### PR TITLE
Use blob url for index file rather than github

### PR DESCRIPTION
**Pull request by bishal-pdMSFT**

### CODE REVIEW

**Comment by bishal-pdMSFT** [comment]
#10969

File: Tasks/UseDotNetV2/versionfetcher.ts [hunk 1]
```diff
@@ -294,4 +293,4 @@ export class DotNetCoreVersionFetcher {
     }
 }
 
-const DotNetCoreReleasesIndexUrl: string = "https://raw.githubusercontent.com/dotnet/core/master/release-notes/releases-index.json";
+const DotNetCoreReleasesIndexUrl: string = "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json";
```
Comment (tejasd1990) [inline Tasks/UseDotNetV2/versionfetcher.ts:296]: should we also make this change in the DotNetCoreInstaller task?

Comment (tejasd1990) [reply Tasks/UseDotNetV2/versionfetcher.ts:296]: minor: this should also be updated in task.json

Comment (bishal-pdMSFT) [reply Tasks/UseDotNetV2/versionfetcher.ts:296]: Done

**Comment by bishal-pdMSFT** [comment]
#12743

**Pull request merged**